services/core/FailoverAgent/failover/agent.py

```diff
@@ -67,6 +67,7 @@
 
 utils.setup_logging()
 _log = logging.getLogger(__name__)
+__version__ = '0.1'
```

Comment:
Probably should be higher since this was out for a bit...Even though I doubt people have used it.